### PR TITLE
Updated styles api and icons

### PR DESCRIPTION
**Pull request by Jin1L**

### CODE REVIEW

**Comment by jackparmer** [comment]
@Jin1L can you please add some screenshots of the changes you're making?

**Comment by Jin1L** [comment]
@jackparmer I am sorry about the inconvenience. 
What I changed:
- Added box-shadow, border for the API Key modal
- Adjusted colours for the button hover and play, settings button
- Adjusted positions of the API modal title & input box
- Adjusted modal box size

The title font looks different on my end. Please let me know if it still looks different.

https://github.com/flojoy-io/studio/assets/89884098/04c6ac74-8ea2-4842-869c-9553fb677c2d

File: src/feature/common/theme.ts [hunk 1]
```diff
@@ -26,10 +26,11 @@ export const darkTheme: MantineThemeOverride = {
     modal: ["#191919", "#111111"],
     title: ["#ffffff"],
     text: ["#bcc2c4"],
-    accent1: ["#99f5ff", "#48abe0"],
+    accent1: ["#99f5ff", "#48abe0", "#d4fbff"],
     accent2: ["#7b61ff", "#7418b5"],
```
Comment (itsjoeoui) [inline src/feature/common/theme.ts:30]: Make sure you update the accent for both theme, not only one.

File: src/feature/common/theme.ts [hunk 2]
```diff
@@ -52,9 +53,10 @@ export const lightTheme: MantineThemeOverride = {
     modal: ["#f4f4f4", "#ffffff"],
     title: ["#111111"],
     text: ["#8c9699"],
-    accent1: ["#2e83ff", "#578bf2"],
-    accent2: ["#7b61ff", "#7418b5"],
+    accent1: ["#2e83ff", "#578bf2", "#d4fbff"],
+    accent2: ["#7b61ff", "#7418b5", "#af9fff"],
     accent3: ["#ffa927"],
     accent4: ["rgba(123, 97, 255, 1)", "rgba(123, 97, 255, 0.17)"], //for add button and leaf nodes
+    accent5: ["#EDEDED", "rgba(237, 237, 237, 0.2)"],
```
Comment (itsjoeoui) [inline src/feature/common/theme.ts:60]: Same thing here

File: src/feature/flow_chart_panel/views/APIKeyModal.tsx [hunk 3]
```diff
@@ -89,14 +114,14 @@ const APIKeyModal = ({ isOpen, onClose }: APIKeyModelProps) => {
         opened={isOpen}
         onClose={handleClose}
         aria-labelledby="API Key modal"
-        size={600}
+        size="calc(303px + 128px)"
```
Comment (JeffDotPng) [inline src/feature/flow_chart_panel/views/APIKeyModal.tsx:117]: You don't need to calc two values with the same units together.

File: src/feature/flow_chart_panel/views/APIKeyModal.tsx [hunk 4]
```diff
@@ -15,41 +15,66 @@ const useStyles = createStyles((theme) => ({
     display: "relative",
     justifyContent: "center",
     alignItems: "center",
+    border:
+      theme.colorScheme === "light"
```
Comment (JeffDotPng) [inline src/feature/flow_chart_panel/views/APIKeyModal.tsx:19]: The exact same value? no need for ternary

File: src/feature/common/theme.ts [hunk 5]
```diff
@@ -26,10 +26,11 @@ export const darkTheme: MantineThemeOverride = {
     modal: ["#191919", "#111111"],
     title: ["#ffffff"],
     text: ["#bcc2c4"],
-    accent1: ["#99f5ff", "#48abe0"],
-    accent2: ["#7b61ff", "#7418b5"],
+    accent1: ["#99f5ff", "#48abe0", "#d4fbff"],
```
Comment (JeffDotPng) [inline src/feature/common/theme.ts:29]: image
This button probably shouldn't be white on hover.

**Comment by jackparmer** [comment]
The API key modal needs to give some kind of feedback after the button is pressed and then when the API key is set. It's hard to tell now whether the API key has been successfully updated after clicking the button:

Can be as simple as a small status message, like:
1. Setting API key...
2. API key updated

**Comment by Jin1L** [comment]
> The API key modal needs to give some kind of feedback after the button is pressed and then when the API key is set. It's hard to tell now whether the API key has been successfully updated after clicking the button:
> 
> Can be as simple as a small status message, like:
> 
> 1. Setting API key...
> 2. API key updated

@jackparmer , I currently implemented notification on the right corner. Do you want me to implement it differently?

https://github.com/flojoy-io/studio/assets/89884098/f87aff44-5e8d-401f-b5f9-dbd3e7474125

File: src/feature/common/theme.ts (hunk 5 above)
Comment (Jin1L) [reply src/feature/common/theme.ts:29]: @JeffDotPng It was the color on flojoy.io/brand for the button hover. Would you like me to change it to a lighter color? Or change the color of the settings button instead?

Comment (JeffDotPng) [reply src/feature/common/theme.ts:29]: Imo just change the hover color to a lighter cyan color like the accent (just lower the opacity on one of the accent colors), this current one is very blinding for dark mode.

Comment (Jin1L) [reply src/feature/common/theme.ts:29]: @JeffDotPng Would this be better color??

Screenshot 2023-05-30 at 8 58 49 PM

Comment (Jin1L) [reply src/feature/common/theme.ts:29]: @JeffDotPng  This is what the setting button looks like right now. Let me know if there is anything else to change!

Screenshot 2023-05-30 at 9 38 31 PM
Screenshot 2023-05-30 at 9 38 42 PM

**Comment by smahmed776** [comment]
I think this might be relevant to this PR. Currently, the disabled play button does not look good.

Screenshot (44)

**Comment by itsjoeoui** [comment]
> > I think this might be relevant to this PR. Currently, the disabled play button does not look good.
> > Screenshot (44)
> 
> @smahmed776 I changed the button color when it's disabled. Let me know if there is anything else to change! Screenshot 2023-05-31 at 10 23 40 AM

I think the button should not be this light in dark mode

**Comment by Jin1L** [comment]
> 

@itsjoeoui , Would this be the better colour?

Screenshot 2023-05-31 at 10 41 47 AM

**Comment by smahmed776** [comment]
> > 
> 
> @itsjoeoui , Would this be the better colour?
> 
> Screenshot 2023-05-31 at 10 41 47 AM
> 

Looks better to me

**Review by smahmed776 — APPROVED** [review]

**Pull request merged**